app/lib/theme/ReadOnlyObjectFieldTemplate.tsx

```diff
@@ -0,0 +1,17 @@
+import Grid from "@button-inc/bcgov-theme/Grid";
+
+const ReadOnlyObjectFieldTemplate = (props) => {
+  return (
+    <>
+      <Grid cols={10}>
+        <Grid.Row>
+          <Grid.Col span={10}>
```

Comment:
What are those elements achieving? would a `div` do the same?

Reply:
They were to match the regular `ObjectFieldTemplate`, but it looks the same with a div so I went with that. I suspect this will get changed as part of showing the diffs anyway to make sure there's space for error messages, etc.